src/workerd/api/crypto/impl.c++

```diff
@@ -220,6 +221,42 @@ void checkPbkdfLimits(jsg::Lock& js, size_t iterations) {
   }
 }
 
+kj::Maybe<kj::Own<BIGNUM>> toBignum(kj::ArrayPtr<const kj::byte> data) {
+  BIGNUM* result = BN_bin2bn(data.begin(), data.size(), nullptr);
+  if (result == nullptr) return kj::none;
+  return kj::Own<BIGNUM>(result, workerd::api::SslDisposer<BIGNUM, &BIGNUM_free>::INSTANCE);
+}
+
+BIGNUM* toBignumUnowned(kj::ArrayPtr<const kj::byte> data) {
+  return BN_bin2bn(data.begin(), data.size(), nullptr);
+}
+
+kj::Maybe<kj::Array<kj::byte>> bignumToArray(const BIGNUM& n) {
+  auto result = kj::heapArray<kj::byte>(BN_num_bytes(&n));
+  if (BN_bn2bin(&n, result.begin()) != result.size()) return kj::none;
+  return kj::mv(result);
+}
+
+kj::Maybe<kj::Array<kj::byte>> bignumToArrayPadded(const BIGNUM& n) {
+  auto result = kj::heapArray<kj::byte>(BN_num_bytes(&n));
+  if (BN_bn2binpad(&n, static_cast<unsigned char*>(result.begin()),
```

Comment:
I wonder whwhy cast is needed here? kj::byte is already unsigned char, isn't it?